Makefile

```diff
@@ -5,12 +5,25 @@ OT1_VERSION := 2.5.2
 OT2_VERSION_TAG := v3.3.1-beta.0
```

Comment:
Why??

Reply:
It was to add in module support asap so we could send off protocols to customers. Can probably stand to be changed.

Reply:
let's update it in a separate PR :upside_down_face: Any maybe we can use pip now?